File: Uri-Online-Judge/Wellington/Python/Iniciante/1020.py
# -*- coding: utf-8 -*-

idade = input() #ler idade em dias

ano = idade / 365 #calculo
idade %= 365
mes = idade / 30
idade %= 30
dia = idade

print("%i ano(s)" % ano) #imprimir
print("%i mes(es)" % mes)
print("%i dia(s)" % dia)

File: Uri-Online-Judge/Wellington/C:C++/Iniciante/1020.cpp
#include <stdio.h>

int main(){
    int idade;
    
    scanf("%i", &idade);
    
    printf("%i ano(s)\n", idade/365);
    printf("%i mes(es)\n", (idade%365)/30);
    printf("%i dia(s)\n", (idade%365)%30);
    
    return 0;
}
